x/migration/module/cmd/collect_morse_accounts.go

```diff
@@ -270,9 +289,25 @@ func collectInputSupplierStakes(inputState *migrationtypes.MorseStateExport, mor
 	for exportSupplierIdx, exportSupplier := range inputState.AppState.Pos.Validators {
 		supplierAddr := exportSupplier.Address.String()
 
-		// DEV_NOTE: An account SHOULD exist for each actor.
 		if !morseWorkspace.hasAccount(supplierAddr) {
-			return ErrMorseExportState.Wrapf("account not found corresponding to supplier with address %q", supplierAddr)
+			logger.Logger.Warn().
+				Str("supplier_address", supplierAddr).
+				Msg("no account found for supplier")
+
+			// DEV_NOTE: If no auth account was found for this supplier, create a new one.
+			newSupplierAccount := &migrationtypes.MorseAuthAccount{
+				Type: "posmint/Account",
+				Value: &migrationtypes.MorseAccount{
+					Address: exportSupplier.Address,
+					Coins:   []cosmostypes.Coin{},
+				},
+			}
+			if _, _, err := morseWorkspace.addAccount(supplierAddr, newSupplierAccount); err != nil {
+				return fmt.Errorf(
+					"adding supplier account to account balance of address %q: %w",
+					supplierAddr, err,
+				)
+			}
```

Comment:
Why don't we have the != "" check for staked tokens for a supplier?

Reply:
Nice catch - thank you!